src/openrct2/rct1/S4Importer.cpp

```diff
@@ -2505,10 +2506,12 @@ namespace RCT1
         {
             if (_s4.scenario_slot_index == SC_URBAN_PARK && _isScenario)
             {
+                const auto firstRideId = static_cast<ride_id_t>(0);
```

Comment:
If we are naming this parameter anyway, `merryGoRoundId` is a better fit and also illustrates the purpose of the fix better.